### DISCUSSION TITLE
Using gnome-terminal

### DISCUSSION THREAD

**Discussion by havardAasen** [Frederick888/external-editor-revived#71] [Show and tell]
Example of using `gnome-terminal` and `vim`.

`gnome-terminal --wait -- vim "/path/to/temp.eml"`

Used `gnome-terminal` version 3.46.2, at the time of testing.

**Comment by Yannik** [comment]
This works, but opens gnome-temrinal in the background (not focused, behind the mail compose window). Do you have any solution for that @havardAasen ? :-)
